dubbo-common/src/main/java/org/apache/dubbo/common/utils/UrlUtils.java

```diff
@@ -445,6 +451,22 @@ public static boolean isServiceKeyMatch(URL pattern, URL value) {
         return urls.stream().filter(predicate).collect(Collectors.toList());
     }
 
+    public static boolean isConfigure(URL url) {
+        return CONFIGURATORS_CATEGORY.equals(url.getParameter(Constants.CATEGORY_KEY, Constants.DEFAULT_CATEGORY))
+                || OVERRIDE_PROTOCOL.equals(url.getProtocol());
+    }
+
+    public static boolean isRoute(URL url) {

```

Comment:
`public static boolean isRoute(final URL url)`

Reply:
no need to use final.

Reply:
ok.